benefit/data/sheets/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -1230,7 +1230,7 @@
     "CaseFieldID": "appeal",
     "ListElementCode": "appellant.role.description",
     "ID": "appealAppellantRole",
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "OPTIONAL",
```

Comment:
I think this needs to stay Mandatory - if they select 'Other' from the field above, then something needs to be populated in this field?

Reply:
Only if it is mandatory as a business rule, and it falls in the same situation as the other appellant role fields.